tools/local-environment/tests/watcherDoubleUpdate.ts

```diff
@@ -47,6 +48,13 @@ import { sleep } from "../src/utils";
   await tomWatcher.connect();
   await tomWatcher.start();
 
+  const updaterWaiter: Waiter<true> = await waitAgentFailure(n, tom, "updater");
+  const processorWaiter: Waiter<true> = await waitAgentFailure(
+    n,
+    tom,
+    "processor"
+  );
```

Comment:
K agent code looks gtg but last change would be to isolate the processor and updater behavior tests in their own test case (one for watcher and failing home, one for shutting down updater/processor on improper update). Cleaner this way to minimize number of things we test in one test.